ci/Jenkinsfiles/build.groovy

```diff
@@ -262,7 +260,7 @@ def buildUnitTestStage(env) {
 }
 
 void archiveKafkaLogs(namespace, logFile) {
-  sh "kubectl logs ${TEST_KAFKA_POD_NAME} --namespace=${namespace} > ${logFile}"
+  sh "kubectl logs ${TEST_KAFKA_POD_NAME} --namespace=${namespace} > ${logFile} || echo"
```

Comment:
What's the `|| echo` for? If it's to not stop if the previous command returns an error, the traditional way is `|| true`

Reply:
It is to avoid the error indeed, I think I tried the echo way without success, let do another try 🤞.

Reply:
`echo` and `true` both returns a status 0 (ok) so they should be equivalent in terms of error recovery, it's just that there's no need to output a newline.